openmp/libomptarget/src/LegacyAPI.cpp

```diff
@@ -10,15 +10,21 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "OpenMP/OMPT/Interface.h"
```

Comment:
Don't think we should bother adding this to the legacy API stuff because these are just kept around for backwards compatibility.

Reply:
> Don't think we should bother adding this to the legacy API stuff because these are just kept around for backwards compatibility.

Well, downstream still uses it. Since the legacy API is still around upstream, I decided to make the changes here.

Reply:
What do we use it for downstream? All of the legacy API functions should have newer versions that `clang` has emitted for around two years now.

Reply:
> What do we use it for downstream? All of the legacy API functions should have newer versions that `clang` has emitted for around two years now.

legacy flang.

Reply:
I see, makes sense.